librz/asm/p/asm_sh.c

```diff
@@ -1,78 +1,51 @@
 // SPDX-FileCopyrightText: 2010-2011 eloi <limited-entropy.com>
 // SPDX-License-Identifier: LGPL-3.0-only
+// SPDX-FileCopyrightText: 2022 Dhruv Maroo <dhruvmaru007@gmail.com>
```

Comment:
No need to double `SPDX-License-Identifier` and put `SDPX-FileCopyrightText` together